spec/requests/api/v1/targets/create_spec.rb

```diff
@@ -0,0 +1,51 @@
+require 'rails_helper'
+
+describe 'POST api/v1/targets', type: :request do
+  subject { post api_v1_targets_path, params: params, headers: auth_headers, as: :json }
+
+  describe 'POST create' do
+    let(:user)          { create :user }
+    let(:topic)         { create :topic }
+
+    let(:title)           { 'New target' }
+    let(:radius)          { 300 }
+    let(:coordinates)     { [-34.6037389, -58.3837591] }
```

Comment:
What do you think about using a Hash for the coordinates? e.g Hash(longitude: 3423.23423, latitude: 234234)
Then you can do coordinates[:latitude] instead of coordinates[:0]